lib/dynamoid/adapter_plugin/aws_sdk_v2.rb

```diff
@@ -1,5 +1,5 @@
 module Dynamoid
-  module Adapter
+  module AdapterPlugin
```

Comment:
They do match... the module name is `AdapterPlugin` while its directory is `adapter_plugin`, and the class name is unchanged.